demo_modules/chicago-brick-live/.jshintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../.jshintrc",
+  // Enable Function to be used.  
```

Comment:
I looked at switching to safeEval in /lib/eval.js but that creates and evaluates in one shot.  This isn't ideal since the drawing code will not change often (relative to the rending loop interval) .  

I could have:
- modified safeEval to return the sandboxed function
- created a /lib/sandboxFunction.js (or something)
- tricked jshint using Function.apply (which is why jshint doesn't whine about safeEval, I think)
- whitelist Function for chicago-brick-live

I ended up going to the last one but am happy to be convinced it was the wrong choice.